.github/workflows/bahmni_metabase_build_publish.yaml

```diff
@@ -0,0 +1,43 @@
+name: Build and Publish bahmni-metabase Image
+on:
+  push:
+    branches: main
+
+jobs:
+
+  Trivy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Trivy Scan
+        run: ./.github/trivy_scan.sh
+
+  docker-build-publish:
+    name: Docker Build & Publish
+    runs-on: ubuntu-latest
+    needs: Trivy
+    steps:
+      - uses: actions/checkout@v2
+      - name: Verify app version
+        run: |
+          APP_VERSION=$(cat docker/.appversion)
```

Comment:
Please use [this](https://github.com/Bahmni/bahmni-infra-utils/blob/main/setArtifactVersion.sh) script instead of writing it here. 
Refer [this](https://github.com/Bahmni/openmrs-module-bahmniapps/blob/master/.github/workflows/build_publish.yml#L18) for an example.

Reply:
Yes I have updated it , Thank you